### PR TITLE
Adding a can() method to AuthorizationComponent

### PR DESCRIPTION
**Pull request by burzum**
Refs #42

### CODE REVIEW

File: src/Controller/Component/AuthorizationComponent.php [hunk 1]
```diff
@@ -59,6 +59,30 @@ class AuthorizationComponent extends Component
      * @throws \Authorization\Exception\ForbiddenException when policy check fails.
      */
     public function authorize($resource, $action = null)
+    {
+        if ($action === null) {
+            $request = $this->getController()->request;
```
Comment (ADmad) [inline src/Controller/Component/AuthorizationComponent.php]: Should use ->getRequest(), ->request is deprecated.

Comment (burzum) [reply src/Controller/Component/AuthorizationComponent.php]: This plugin is still using 3.5?

Comment (ADmad) [reply src/Controller/Component/AuthorizationComponent.php]: IMO those willing to put in the effort to upgrade there existing projects to using the plugin would also be upgrading to latest cake version too.

Comment (burzum) [reply src/Controller/Component/AuthorizationComponent.php]: I would like to go to 3.6 soon but we have a ton of plugins that need to upgrade as well. :(

Comment (markstory) [reply src/Controller/Component/AuthorizationComponent.php]: I think we should stay 3.5+ for a while longer.

**Comment by robertpustulka** [comment]
Docs need to be updated with the new method.

**Pull request merged**